src/luarocks/cmd.lua

```diff
@@ -455,9 +459,10 @@ Enabling completion for Fish:
       end)
    parser:flag("--dev", "Enable the sub-repositories in rocks servers for "..
       "rockspecs of in-development versions.")
-   parser:option("--server", "Fetch rocks/rockspecs from this server "..
+   parser:option("--server", "Fetch rocks/rockspecs from this servers "..
```

Comment:
I think there is an method in argparse to allow an `option` to be passed multiple times (resulting in an array of values). I think that would be preferrable to relying on splitting by spaces.